framer/SVG.coffee

```diff
@@ -31,6 +31,56 @@ class exports.SVG
 		"""
 		svgLayer.fill = "url(##{id})"
 
+	@updateImagePatternSVG: (svgLayer) ->
+		return if svgLayer.__constructor
+
+		if not svgLayer.image
+			svgLayer._elementImagePatternSVG?.innerHTML = ""
+			return
+
+		transform = ""
+
+		if svgLayer.backgroundSize in ["fill", "fit", "contain", "cover"] and svgLayer.imageSize
+			scaleX = 1
+			scaleY = 1
+			offsetX = 0
+			offsetY = 0
+
+			imageWidth = svgLayer.imageSize.width
+			imageHeight = svgLayer.imageSize.height
```

Comment:
Imagesize can be null

Reply:
I check if imageSize isn't null on line 43